data-prepper-plugin-schema/src/test/java/org/opensearch/dataprepper/schemas/JsonSchemaConverterTest.java

```diff
@@ -65,6 +77,12 @@ static class TestConfig {
         @JsonProperty(defaultValue = "default_value")
         private String testAttributeWithDefaultValue;
 
+        @JsonProperty
+        @IfPresentAlsoRequire(values = {"test_mutually_exclusive_attribute_b:null"})
```

Comment:
What is this `:null` part for? Should we have a distinct model to represent the additional requirement?

e.g.

```
@IfPresentAlsoRequire({@Required(name = "test_mutually_exclusive_attribute", allowedValues = {null}))
```